festflow/admin.py

```diff
@@ -35,6 +35,7 @@ class NewsLetterAdmin(admin.ModelAdmin):
 
 admin.site.register(Profile)
 admin.site.register(Event)
+admin.site.register(Keynote)
```

Comment:
:P Not an issue, but you could just add it at the end, right? this helps keeping git blame clear.